drivers/backup/auth.go

```diff
@@ -944,6 +972,42 @@ func DeleteGroup(group string) error {
 	return nil
 }
 
+// Deletes Multiple groups
+func DeleteMultipleGroup(groups []string) error {
+
+	var wg sync.WaitGroup
+	for _, group := range groups {
+		wg.Add(1)
+		go func(group string) {
+			err := DeleteGroup(group)
+			log.FailOnError(err, "Failed to create group - %v", group)
+			wg.Done()
```

Comment:
At all the places where you are using go routines male add "defer wg.Done()" and remove it here

Reply:
`go func(group string) {`
			`defer wg.Done()`
			`err := DeleteGroup(group)`
			`log.FailOnError(err, "Failed to create group - %v", group)`
		`}`